IO-pc-local/src/app/lobby/lobby.component.ts

```diff
@@ -21,26 +21,46 @@ import { Router } from '@angular/router';
 })
 
 
-export class LobbyComponent {
+export class LobbyComponent implements OnInit{
+  private lobbyService = inject(LobbyService);
+  private router = inject(Router);
+  private cdr = inject(ChangeDetectorRef);
 
-  public students$: Observable<Student[]>;
+  students = [] as Student[];
 
-  constructor(private router: Router,private lobbyService: LobbyService){
-    this.students$ = lobbyService.students$;
+  constructor(){
+    // this.students$ = this.lobbyService.students$;
```

Comment:
Usuń zakomentowany kod albo dodaj komentarz tłumaczący, dlaczego jest zostawiony

Reply:
Ogólnie to ja często zapominam po sobie posprzątać także dzięki za zwrócenie uwagi